cli/src/cmd/forge/test.rs

```diff
@@ -438,29 +438,45 @@ fn short_test_result(name: &str, result: &TestResult) {
     let status = if result.success {
         Paint::green("[PASS]".to_string())
     } else {
-        let txt = match (&result.reason, &result.counterexample) {
-            (Some(ref reason), Some(ref counterexample)) => {
+        let txt = match (&result.reason, &result.counterexample, &result.counterexample_sequence) {
+            (Some(ref reason), Some(ref counterexample), None) => {
                 format!("[FAIL. Reason: {reason}. Counterexample: {counterexample}]")
             }
-            (None, Some(ref counterexample)) => {
+            (None, Some(ref counterexample), None) => {
                 format!("[FAIL. Counterexample: {counterexample}]")
             }
-            (Some(ref reason), None) => {
+            (Some(ref reason), None, None) => {
                 format!("[FAIL. Reason: {reason}]")
             }
-            (None, None) => "[FAIL]".to_string(),
+            (Some(ref reason), None, Some(sequence)) => {
+                let mut inner_txt = "".to_string();
+
+                for checkpoint in sequence {
+                    inner_txt += format!("{checkpoint}\n").as_str();
+                }
+
+                format!("###\nFAIL. Reason: {reason}. | Sequence:\n{inner_txt}\n")
```

Comment:
Why is the formatting different here? I think we should probably go for something like:

```
[FAIL. Reason: <reason>] <test or whatever we have now>
<checkpoints>
<sequence>
```

Alternatively we should move to a different format altogether to keep it consistent